sources/packages/api/src/database/entities/offering.type.ts

```diff
@@ -8,7 +8,11 @@ export enum OfferingTypes {
   public = "Public",
   /**
    * Offering was created to fulfill the need of
-   * a particular student application.
+   * a particular student/application.
    */
-  applicationSpecific = "Application Specific",
+  Private = "Private",
+  /**
+   * Offering created for a change in scholastic standing reported by institution.
+   */
+  ScholasticStanding = "Scholastic Standing",
 }
```

Comment:
Which one are we sticking to ? I see public in line 8 begins with lower case while the other ones (Private and ScholasticStanding) begin with upper case.

Reply:
Valid point. I will change public to **P**ublic. It is old code before we made such decisions and missed to be refactored.